paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -49,6 +49,17 @@ class PaparazziPluginTest {
     assertThat(result.output).contains("platforms/android-oops")
   }
 
+  @Test
+  fun applyOutOfOrder() {
```

Comment:
Creates paparazzi-gradle-plugin/src/test/projects/apply-out-of-order/src/test/snapshots/images/app.cash.paparazzi.plugin.test_VerifyTest_verify.png
which is not in gitignore, what to do?

Reply:
see `missingPlugin` for the template to use here.

task = preparePaparazziDebugResources

Reply:
```suggestion
  fun applyAndroidPluginAfter() {
```